shared/people/index.shared.tsx

```diff
@@ -67,15 +68,43 @@ const EmailVerificationBanner = ({email}) => {
       () => dispatch(SignupGen.createClearJustSignedUpEmail()),
     []
   )
+  if (email === 'none') {
+    return <Kb.Banner color="green">{`Welcome to Keybase!`}</Kb.Banner>
+  }
   return (
     <Kb.Banner color="green">{`Welcome to Keybase! A verification link was sent to ${email}.`}</Kb.Banner>
   )
 }
 
+const ResentEmailVerificationBanner = ({email}) => {
+  const dispatch = Container.useDispatch()
+  React.useEffect(
+    () =>
+      // Only have a cleanup function
+      () =>
+        dispatch(
+          PeopleGen.createSetResentEmail({
+            email: '',
+          })
+        ),
+    []
+  )
+
+  return (
+    <Kb.Banner color="yellow">
+      <Kb.BannerParagraph
+        bannerColor="yellow"
+        content={`Check your inbox! A verification link was sent to ${email}.`}
+      />
+    </Kb.Banner>
+  )
+}
+
 export const PeoplePageList = (props: Props) => (
   <Kb.Box style={{...Styles.globalStyles.flexBoxColumn, position: 'relative', width: '100%'}}>
     {Styles.isMobile && <AirdropBanner showSystemButtons={false} />}
     {!!props.signupEmail && <EmailVerificationBanner email={props.signupEmail} />}
+    {!!props.resentEmail && <ResentEmailVerificationBanner email={props.resentEmail} />}
```

Comment:
Can `ResentEmailVerificationBanner` get `resentEmail` out of the store and return null if not set?